tests/js/schemas.js

```diff
@@ -44,8 +44,15 @@ PersonObject.prototype.description = function() {
 PersonObject.prototype.toString = function() {
     return this.name;
 };
-Object.setPrototypeOf(PersonObject, Realm.Object);
-Object.setPrototypeOf(PersonObject.prototype, Realm.Object.prototype);
+
+// Object.setPrototypeOf doesn't work on JSC on Android. The code below achieves the same thing.
+//Object.setPrototypeOf(PersonObject, Realm.Object);
+//Object.setPrototypeOf(PersonObject.prototype, Realm.Object.prototype);
+
+PersonObject.__proto__ = Realm.Object.__proto__;
```

Comment:
The right-hand part shouldn't use `__proto__`.
Should be just `PersonObject.__proto__ = Realm.Object`.